chromium/preloadlist/preloadlist.go

```diff
@@ -47,6 +47,9 @@ type PreloadList struct {
 	Entries []Entry `json:"entries"`
 }
 
+// PolicyType represents the policy under which the domain was added to the preload list
+type PolicyType string 
+
```

Comment:
This PR would a good place to list out the possible values for a PolicyType.

Reply:
Are the possible  values only "bulk-18-weeks", "bulk-1-year", and "custom"?

Reply:
By list out, I meant constants, e.g.:

```
const (
        // test domains.
        PolicyTest PolicyType = "test"
        // Google-owned sites.
        PolicyGoogle = "google"
        // ...
)
```

Reply:
Done